Docs: Update Contributing Files (#1311)
longturn/freeciv21: docs/Contributing/bugs.rst

* Check the Freeciv21 Releases page at https://github.com/longturn/freeciv21/releases, to ensure you're
  playing the latest version. We may have already fixed the problem.

* If the first two items do not work, then feel free to submit a new issue. We have implemented GitHub issue

>   templates so they will walk you through the major aspects of a new issue. It is important to note that the

Reviewer: ```suggestion
  templates which will walk you through the major aspects of a new issue. It is important to note that the
```

>   more details you can provide, the better. If a screen shot is available, then by all means supply it to us.

Reviewer: ```suggestion
  more details you can provide, the better. If a screenshot is available, then by all means add it to the issue.
```
